core/server/master/src/main/java/alluxio/master/file/FileSystemMasterAuditContext.java

```diff
@@ -35,6 +35,7 @@
   private AuthType mAuthType;
   private String mIp;
   private Inode mSrcInode;
+  private long mExecutionTime;
```

Comment:
CreationTime. Originally i thought you are going to store the time spent in a context too and I was suggesting to use ExecutionTime for that value.

```suggestion
  private long mCreationTime;
```